usaspending_api/awards/v2/views/idvs/funding_accounts.py

```diff
@@ -115,23 +115,17 @@ class IDVFundingAccountViewSet(IDVFundingBaseViewSet):
     @cache_response()
     def post(self, request: Request) -> Response:
         order_by = SQL("order by {} {}".format(SORTABLE_COLUMNS[request.data['sort']], request.data['order']))
-        group_by = SQL("""group by federal_account, fa.account_title, cte.award_id, ta.abbreviation, ta.name, a.id""")
+        group_by = SQL("""group by federal_account, fa.account_title, ta.abbreviation, ta.name, a.id""")
         columns = SQL('''sum(nullif(faba.transaction_obligated_amount, 'NaN'::numeric)) total_transaction_obligated_amount,
             taa.agency_id || '-' || taa.main_account_code federal_account,
-            sum(sum(nullif(faba.transaction_obligated_amount, 'NaN'::numeric))) over (partition by cte.award_id) as
-            total,
```

Comment:
No summing at all? I have no idea what this area of code is doing, but that seems odd to me...

Reply:
I mean, this is specifically what this hotfix was designed to remove: the bad grouping and summing.  The total is presented accurately elsewhere (in another endpoint).